@navikt/core/css/chips.css

```diff
@@ -20,7 +20,7 @@
   cursor: pointer;
   align-items: center;
   justify-content: center;
-  gap: var(--a-spacing-1);
+  gap: var(--a-spacing-05);
```

Comment:
https://www.chromatic.com/test?appId=5f801fb2aea7820022de2936&id=652ff042f1aa202996a246cc
> Sirkel ser visuelt nærmere ut her enn X ikonet i removable. Bør det være noen px ekstra gap?

Reply:
Har spilt det inn for Sjur, men ser at denne gap-en gjør at Removable og Toggle tar like stor plass 🤓

Reply:
"det skal være litt mindre avstand mellom ikonet og teksten enn ikonet og venstre kant. Også kan det være mange toggles i en gruppe, så hver px teller"